docs/users/fbc_workflow.md

```diff
@@ -0,0 +1,154 @@
+# FBC workflow
+
+The prerequisite for the File Based Catalog workflow is to convert the existing operator
+to FBC format. This can be done by following a [onboarding](./fbc_onboarding.md) documentation.
+
+After an existing operator is onboard or you introduce a brand new operator you can
+start the FBC workflow.
+
+## FBC templates
+File-based catalog templates serve as a simplified view of a catalog that can be updated
+by the user. The OPM currently supports 3 types of templates and it is up to the user which
+template the operator will be using.
+
+* Basic template
+* SemVer template
+* Composite template
+
+More information about each template can be found at [opm doc](https://olm.operatorframework.io/docs/reference/catalog-templates/).
+
+The recommended template from the maintainability point of view is `SemVer` or `Composite` with
+`SemVer`integration.
+
+## Generate catalogs using templates
+To generate a final catalog for an operator a user needs to execute different `opm`
+commands based on the template type. We as operator pipeline maintainers want
+to simplify this process and we prepared a `Makefile` with all pre-configured targets.
+
+To get the `Makefile` follow these steps
+
+```bash
+cd <operator-repo>/operator/<operator-name>
+wget https://raw.githubusercontent.com/redhat-openshift-ecosystem/operator-pipelines/main/fbc/Makefile
+```
+
+The right place for the Makefile is in the operator's root directory
+```
+.
+├── 0.0.1
+│   ├── manifests
+│   └── metadata
+├── catalogs.yaml
+├── catalog-templates
+├── ci.yaml
```

Comment:
I saw no mention of this file in the previous onboarding doc.  I know in my prototype I presumed that we'd want to update this file to clearly identify that this operator had been onboarded to FBC.  How does the pipeline know who has been onboarded?

Reply:
yeah, the fbc onboarding automatically sets the values in the ci.yaml file and I just added this point also to the doc.